pkg/load/load.go

```diff
@@ -32,6 +47,48 @@ func Config(path string) (*api.ReleaseBuildConfiguration, error) {
 	if err := yaml.Unmarshal([]byte(raw), configSpec); err != nil {
 		return nil, fmt.Errorf("invalid configuration: %v\nvalue:\n%s", err, raw)
 	}
+
+	// If configresolver address info is provided, GET a copy of the config from there
+	if info != nil {
+		req, err := http.NewRequest("GET", fmt.Sprintf("http://%s/config", info.Address), nil)
```

Comment:
Personally I would consider putting the gory http request details into a function for readability, but I do not insist on it.